app/assets/stylesheets/components/header.scss

```
@@ -124,22 +124,29 @@
 
 .skip-content-link {
   position: absolute;
-  top: 0;
   left: 50%;
-  transform: translate(-50%, -100px);
   background: var(--body-color);
   color: var(--body-color-inverted);
   z-index: var(--z-popover);
   padding: var(--su-4);
   pointer-events: none;
-  opacity: 0;
   border-radius: var(--radius);
-  outline: 0;
+  height: var(--header-height);
+  transform: translate(-50%, calc(-1 * var(--header-height)));
+  transition: transform 0.1s ease-in;
```

Maybe we could use our standard transition properties here? No strong opinion though.

```suggestion
  transition: transform var(--transition-props);
```

Visually basically no difference but it would be more consistent in code.